docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+---
+---
+
+Kubernetes ships with a default scheduler that is described [here](kube-scheduler.md).
+If the default scheduler does not suit your needs you can implement your own scheduler.
+Not just that, you can even run multiple schedulers simultaneously alongside the default
+scheduler and instruct Kubernetes what scheduler to use for each of your pods. Let's
+learn how to run multiple schedulers in Kubernetes with an example.
+
+A detailed description of how to implement a scheduler is outside the scope of this
+document. Please refer to the kube-scheduler implementation in
+[plugin/pkg/scheduler](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler)
+in the Kubernetes source directory for a canonical example.
+
+### 1. Package the scheduler
+
+Package your scheduler binary into a container image. For the purposes of this example,
+let's just use the default scheduler (kube-scheduler) as our second scheduler as well.
+Clone the [Kubernetes source code from Github](https://github.com/kubernetes/kubernetes)
+and build the source.
+
+```shell
+$ git clone https://github.com/kubernetes/kubernetes.git
+$ cd kubernetes
+$ hack/build-go.sh
+```
+
+Create a container image containing the kube-scheduler binary. Here is the `Dockerfile`
+to build the image:
+
+```Dockerfile
```

Comment:
[Pygments](http://pygments.org/) that [rougue](https://github.com/jneen/rouge) is inspired from already supports [Dockerfile](http://pygments.org/docs/lexers/). And there is an PR [https://github.com/jneen/rouge/pull/337] in progress to add Dockerfile support to rogue.

I would just leave Dockerfile here, when rogue gets the support it will render it correctly.